syntax = "proto3";
package MSG;
message msg_people
{
    string name = 1;
    int32 age = 2;
}
message msg_address
{
    string street = 1;
    string city = 2;
}

message WrapperMessage {
    string topic = 1;
    uint64 timestamp = 2;
    oneof message_type {
        msg_people people = 3;
        msg_address address = 4;
    }
}